### ISSUE TITLE
Direct links for uploaded compounds

### ISSUE THREAD

**Issue by rsanchezgarc** [m2ms/fragalysis-frontend#702]
Suggested here https://github.com/xchem/fragalysis-frontend/issues/270

**Comment by tdudgeon** [comment]
Reading https://github.com/xchem/fragalysis-frontend/issues/270 it seems that this issue now only relates to linking to uploaded compound sets (RHS). @rsanchezgarc is this correct?
Assuming so, what is the objective - to be able to link to an uploaded file, or to be able to link to (download) individual molecules in those files?

**Comment by boriskovar-m2ms** [comment]
I don't think this has anything to do with downloading things. This is about displaying stuff where you can just have url in which is encoded what should be displayed in fragalysis without the use of snapshots. Well in the future (when download structures dialog is extended for downloading RHS stuff) this can be used to download things in indirect way (in download structures dialog there is an option to download visible stuff).

@rsanchezgarc is this correct? Because this is what direct links for LHS are doing/used for.

**Comment by rsanchezgarc** [comment]
You are right @boriskovar-m2ms, it is about the links displaying the GUI

**Comment by phraenquex** [comment]
Will be in #858
